shared/xccdf/system/accounts/physical.xml

```diff
@@ -252,6 +252,40 @@ access when the system is rebooted.
 <!--ref nist="IA-2(1),AC-3" disa="213" /-->
 </Rule>
 
+<Rule id="disable_ctrlaltdel_ctrlaltdelburstaction" severity="high" prodtype="rhel7">
```

Comment:
Please make all the script filenames as well as rule/oval names as:
`disable_ctrlaltdel_burstaction`